vagrant/Vagrantfile

```diff
@@ -50,6 +50,8 @@ Vagrant.configure("2") do |config|
               echo "module.sig_unenforce" | sudo tee /etc/kernel/cmdline.d/allow-unsigned-modules.conf
               echo "intel_iommu=igfx_off" | sudo tee /etc/kernel/cmdline-removal.d/disable-iommu.conf
               clr-boot-manager update
+              sudo mkdir -p /etc/systemd/resolved.conf.d
+              printf "[Resolve]\nDNSSEC=false" | sudo tee /etc/systemd/resolved.conf.d/dnssec.conf
```

Comment:
Is this necessary? Clear's systemd is built using `-Ddefault-dnssec=no`

Reply:
Well, maybe it's related to the [Intel's network](https://github.com/AntonioMeireles/vagrant-guests-clearlinux/issues/3#issuecomment-494928154) but if I don't manually disable I get timeouts.

Reply:
Does the vagrant image have the latest systemd from Clear?

Reply:
I had some [issues](https://github.com/AntonioMeireles/vagrant-guests-clearlinux/issues/3) before trying to use the [latest CLR image](https://app.vagrantup.com/AntonioMeireles/boxes/ClearLinux) but maybe I can give another try

Reply:
Did it work?

Reply:
Nope :( I'm going to keep trying to see if I can make it work.

Reply:
Perhaps not spend too much time on it now. It can be dropped later if it works.